Lombiq.Tests.UI/Services/WebDriverFactory.cs

```diff
@@ -11,82 +9,85 @@
 
-                configuration.BrowserOptionsConfigurator?.Invoke(chromeConfig.Options);
+            configuration.BrowserOptionsConfigurator?.Invoke(chromeConfig.Options);
 
-                chromeConfig.Service = driverPath == null
-                    ? ChromeDriverService.CreateDefaultService()
-                    : ChromeDriverService.CreateDefaultService(driverPath);
+            chromeConfig.Service = ChromeDriverService.CreateDefaultService();
 
-                chromeConfig.Service.SuppressInitialDiagnosticInformation = true;
-                // By default localhost is only allowed in IPv4.
-                chromeConfig.Service.AllowedIPAddresses += "::ffff:127.0.0.1";
-                // Helps with misconfigured hosts.
-                if (chromeConfig.Service.HostName == "localhost") chromeConfig.Service.HostName = "127.0.0.1";
+            chromeConfig.Service.SuppressInitialDiagnosticInformation = true;
+            // By default localhost is only allowed in IPv4.
+            chromeConfig.Service.AllowedIPAddresses += "::ffff:127.0.0.1";
+            // Helps with misconfigured hosts.
+            if (chromeConfig.Service.HostName == "localhost") chromeConfig.Service.HostName = "127.0.0.1";
 
-                configuration.Arguments.SetItems(chromeConfig.Options.Arguments);
+            configuration.Arguments.SetItems(chromeConfig.Options.Arguments);
 
-                return new ChromeDriver(chromeConfig.Service, chromeConfig.Options, pageLoadTimeout)
-                    .SetCommonTimeouts(pageLoadTimeout);
-            });
-
-        var chromeWebDriverPath = Environment.GetEnvironmentVariable("CHROMEWEBDRIVER");
-        if (chromeWebDriverPath is { } driverPath && Directory.Exists(driverPath))
-        {
-            return CreateDriverInnerAsync(driverPath);
-        }
-
-        return CreateDriverAsync(BrowserNames.Chrome, () => CreateDriverInnerAsync());
-    }
+            return new ChromeDriver(chromeConfig.Service, chromeConfig.Options, pageLoadTimeout)
+                .SetCommonTimeouts(pageLoadTimeout);
+        }));
 
     public static Task<Func<EdgeDriver>> CreateEdgeDriverAsync(BrowserConfiguration configuration, TimeSpan pageLoadTimeout) =>
-        CreateDriverAsync<EdgeDriver>(BrowserNames.Edge, async () =>
+        CreateDriverAsync(() =>
         {
             var options = new EdgeOptions().SetCommonOptions();
 
             options.SetCommonChromiumOptions(configuration);
 
-            // While the Edge driver easily locates Edge on Windows, it struggles on Linux, where the different release
-            // channels have different executable names. This setting looks up the "microsoft-edge-stable" command and
-            // sets the full path as the browser's binary location.
-            if (!OperatingSystem.IsWindows() &&
-                (await CliWrapHelper.WhichAsync("microsoft-edge-stable"))?.FirstOrDefault() is { } binaryLocation)
```

Comment:
This and the rest of the browser config/driver shenanigans don't seem to be needed when using Selenium Manager. Could you please check @sarahelsaig whether you have any issues running `MultiBrowserTests` under Linux? This all seems to be working as advertised in CI.

Reply:
@sarahelsaig ping.

Reply:
I've tried running `MultiBrowserTests` in this branch, works fine for me. I've seen all the browsers appear one after the other.